src/engine/srv.c

```diff
@@ -1242,11 +1247,40 @@ dss_srv_fini(bool force)
 	return 0;
 }
 
+static int
+dss_sys_db_init(struct sys_db **db)
+{
+	int	 rc;
+	bool	 md_on_ssd_enabled;
+	/* walkaround*/
+	char	*lmm_db_path = getenv("DAOS_LMM_DB_PATH");
+
+	d_getenv_bool("DAOS_MD_ON_SSD", &md_on_ssd_enabled);
```

Comment:
Is this a temporary workaround? I think we'd decide using vos sysdb when system is in old format, right? (We could tell it via the parameters passed from control plane? Need to confirm this with control plane team).

Reply:
Yup, this is a walkaround for me to testing.

Reply:
Ok. Let's make it in a follow-on patch.

I think we might need to parse the passed in daos_nvme.conf to see if meta SSD is configured, see dss_xstream_has_nvme().